### PR TITLE
Update /create-club servlet to add ownership to club

### LINKED ISSUE #62: Update /create-club servlet to add ownership to club
Call the `addOwnership` method in the Storage Handler API upon the creation of a club in the /create-club servlet.

### PR DESCRIPTION
**Pull request by linamontes10-zz**
Calls the `addOwnership` method in the Storage Handler API upon the creation of a club in the /create-club servlet. This method creates an entry in the Memberships table with a membershipType of owner.
Closes #62 .

### CODE REVIEW

File: server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java [hunk 1]
```diff
@@ -50,16 +51,19 @@
...
   public static final String LOG_INPUT_ERROR_MESSAGE =
       "Error with JSON input in CreateClubServlet: ";
+  private final StorageHandlerApi handler;
   private static IdentifierGenerator idGen;
   private static final Gson gson = new Gson();
 
   /** 
    * Overloaded constructor for dependency injection.
    * @param idGen the {@link IdentifierGenerator} that is used when constructing the Club
+   * @param handler the {@link StorageHandlerApi} that is used when saving the Club
```
Comment (JosephBushagour) [inline server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java]: Think about switching the order of these @params, right now they imply the opposite order

**Review by JosephBushagour — APPROVED** [review]
Looks great!

File: server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java (hunk 1 above)
Comment (linamontes10-zz) [reply server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java]: Got it!

**Review by KeatonTech — APPROVED** [review]

**Review by SarahGB — APPROVED** [review]

**Review by jimmycolema — APPROVED** [review]
lgtm!

**Pull request merged**